EIPS/eip-draft-smartcontract-fee-share.md

```diff
@@ -0,0 +1,18 @@
+---
+eip: <to be assigned>
+title: Smart Contract Deployer Transactions Fee Share
+author: @tarekskr
+status: Draft
+type: Standards Track
+category: Core
+created: 2021-02-20
+---
+
+## Simple Summary
+This EIP proposes that a smart contract deployer should get a share of all transaction fees generated by the smart contract.
+
+## Motivation
+A lot of smart contract developers are not able to capture any value from their widely used smart contracts, and are thus forced to deploy meaningless tokens in order to be able to do so. If developers get a small share of all the fees generated by users interacting with their smart contracts, it will potentially solve the aforementioned problem, in addition to providing huge incentives for developers to choose Ethereum over other platforms for their applications.
+
```

Comment:
Missing specification, rationale, and security considerations sections.

Reply:
> Missing specification, rationale, and security considerations sections.
Would really love any help I could get with this 🙏

Reply:
Every EIP needs a technical champion.  Keep in mind, EIPs are meant to be technical standards, not governance discussions.  If you just want to get the discussion going about this idea I recommend starting a thread over at https://ethresear.ch/ or https://ethereum-magicians.org/ rather than starting with an EIP.

Reply:
@MicahZoltu oh I see, thank you so much for the clarification. I will start a thread on https://ethereum-magicians.org/. 

I'd also be really grateful if you have any recommendations on who would be interested to be a technical champion for this.